static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,66 +1,47 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
```

Comment:
if you don't mind I'll do it [there](https://github.com/guardian/frontend/pull/11533).